api/route/router.ts

```diff
@@ -38,7 +38,7 @@ export default function createRouter(namespace: string = '', operations: Operati
     return router;
   }, Router());
 }
-export function useRouter(instance: ExpressInstance, namespace: string = '/', router: Router, authenticated: boolean = false) {
+export function useRouter(instance: ExpressInstance, router: Router, authenticated: boolean = false) {
   if (authenticated) {
     instance.use('/', authRoute, router);
   }
```

Comment:
no need to check for `authenticated`:

```ts
/* if (authenticated) */ instance.use.('/', authRoute, router);
// else instance.use('/', router);
```

Your middleware calls `next` if the route has no associated permission (in the database), so we may as well run all routes though the middleware.